util/test/perf/perfgraph.js

```diff
@@ -118,6 +118,14 @@ var branchInfo = [
                   { "release" : "1.24",
                     "releaseDate": "2021-03-18",
                     "branchDate" : "2021-03-11",
+                    "revision" : -1},
+                  { "release" : "1.24.1",
+                    "releaseDate": "2021-04-15",
+                    "branchDate" : "2021-03-11",
```

Comment:
Branch date looks too early here

Reply:
Right because we built the 1.24.1 release off of the 1.24 branch instead of making a new 1.24.1 branch

Reply:
I think this is more "when did we cut/update the branch" not just when was it created. The `branchDate` is what we turn into the yellow lines in the graphs to mark release timeframes since that reflects when code changes that will impact performance are finished.